### DISCUSSION TITLE
How to analyze existing plots to make sure they are statistically no difference?

### DISCUSSION THREAD

**Discussion by pluto2021** [Chia-Network/chia-blockchain#4250] [Ideas]
Supposedly the plots carries some sort of randomness in their chance of winning. Besides checking the plots one by one, is there any possibility to do a fairness evaluation of all the plots and outputs some sort of plot counts vs. probability of wining chart? Supposedly it should be a flat line, but would like to see the data, or some portion of the plots will have lower probability of winning?

**Comment by cccat6** [comment]
Don't try to use statistics to analyze this. The only thing you need to do is read through the green paper. Math can prove it. There is no difference between different plots. Even they do have some differences. Unless you own most of the network. You won't be able to analyze it because you have too few samples.
